serveradmin/serverdb/models.py

```diff
@@ -29,7 +30,7 @@
     'relation': str,
     'reverse': str,
     'number': lambda x: float(x) if '.' in str(x) else int(x),
-    'inet': lambda x: ip_network(x) if '/' in str(x) else ip_address(x),
+    'inet': lambda x: inet_to_python(x),
```

Comment:
Note that this was doing something else than `inet_to_python()`

`IPInterface` objects are something difficult for people to use.  They allow host addresses with network length or network addresses with host bits set, like `10.0.0.1/24`.  We allow either a host address like `10.0.0.1/32`, or a network address like `10.0.0.0/24` when `ip_addr_type` is `network`.

Postgres `inet` data type works just like `IPInterface`, so we could allow these without any extra effort, but chose to restrict things having host or network addresses explicitly to keep things more structured.

Reply:
Thanks for pointing this out again. I am aware that this is a change of behaviour in the method but I wanted to have one transformation method for inet and intern_ip attribute(s) and networks and hosts. The related logic for validation ensures that a host remains a host and a network a network. The overall behaviour for this should be the same if I did not miss any tests ;-)